piecewise_web/index.html

```diff
@@ -401,17 +406,28 @@ <h3 class="league-gothic">Thank you!</h3>
 map.scrollWheelZoom.disable();
 var control = L.control.zoom({position: 'topright'});
 map.addControl(control);
-var osmLayer = L.tileLayer('http://{s}.tile.osm.org/{z}/{x}/{y}.png', {
-	attribution: '&copy; <a href="http://osm.org/copyright">OpenStreetMap</a>' +
-		'contributors'
-});
 
+// Use Open Street Maps as a base tile layer
+// var osmLayer = L.tileLayer('http://{s}.tile.osm.org/{z}/{x}/{y}.png', {
+//	attribution: '&copy; <a href="http://osm.org/copyright">OpenStreetMap</a>' +
+//		'contributors'
+// });
+
+// Use Mapbox as a base tile layer
+// var mapboxLayer = L.tileLayer(
+//		'https://{s}.tiles.mapbox.com/v3/newamerica.lcl1jan5/{z}/{x}/{y}.png', {
+//	attribution: '&copy; <a href="http://mapbox.com/">Mapbox</a>'
+// });
+ 
+// Use Mapbox as a base tile layer
 var mapboxLayer = L.tileLayer(
 		'https://{s}.tiles.mapbox.com/v3/newamerica.lcl1jan5/{z}/{x}/{y}.png', {
 	attribution: '&copy; <a href="http://mapbox.com/">Mapbox</a>'
 });
 
-// Set the default base tile layer
+// Set the default base tile layer. 
+// If using Open Street Maps: map.addLayer(osmLayer);
```

Comment:
Okay, good. I just wanted to clarify that.  But just one more comment on my thinking, my main thinking  in the earlier comment about the the commented out "osmLayer" and this comment above is that I think this could be simplified by not commenting out the "osmLayer" declaration above, and simply instructing users to change the below (`map.addLayer(mapboxLayer)`) with the desired tile provider.  That is, if you want Mapbox maps set it to mapboxLayer, and if you want OSM maps, set it to osmLayer.  It adds extra work (and documentation) to have users comment and uncomment the layer definitions earlier whenever they want to make a change. I don't think there is any performance hit in defining a bunch of tile providers, and users can just apply the one they want below without having to uncomment anything earlier in the file.